packages/tenant-process/src/services/tenantService.ts

```diff
@@ -1712,6 +1713,43 @@ export function tenantServiceBuilder(
         )
       );
     },
+    async assignTenantDelegatedConsumerFeature({
+      correlationId,
+      organizationId,
+      logger,
+    }: {
+      correlationId: CorrelationId;
+      organizationId: TenantId;
+      logger: Logger;
+    }): Promise<void> {
+      logger.info(
+        `Assigning delegated consumer feature to tenant ${organizationId}`
+      );
+
```

Comment:
As discussed at the weekly today, add IPA check here so that we enforce that delegates are IPA - producer delegations already have the same check, see: https://vscode.dev/github/pagopa/interop-be-monorepo/blob/develop/packages/tenant-process/src/services/tenantService.ts#L1648